docs/src/integrations/exchange.md

```diff
@@ -488,10 +488,121 @@ You can also doublecheck whether a particular blockhash is still valid by sendin
+### Installing the `spl-token` CLI Tool
+
+`spl-token` is distributed from Rust [crates.io](https://crates.io) via the Rust
+`cargo` command line utility. The latest version of `cargo` can be installed
+using a handy one-liner for your platform at [rustup.rs](https://rustup.rs). Once
+`cargo` is installed, `spl-token` can be obtained with the following command:
+
+```
+cargo install spl-token-cli
+```
+
+You can then check the installed version to verify
+
+```
+spl-token --version
+```
+
+Which should result in something like
+
+```text
+spl-token-cli 2.0.1
+```
+
+### Account Creation
+
+SPL Token accounts impose additional requirements as opposed to native System
+Program accounts.
+
+1. SPL Token accounts are not implicitly created, so must be created explicitly
+before an SPL Token balance can be deposited
+1. SPL Token accounts must remain [rent-exempt](https://docs.solana.com/apps/rent#rent-exemption)
+for the duration of their existence and therefore require a small amount of 
+native SOL tokens be deposited at account creation. For SPL Token v2 accounts,
+this amount is 2,039,280 Lamports (0.00203928 SOL).
+
+When a user wishes to deposit a new SPL Token type, a new account will need to be
+created for that user at the given mint.
+
+#### Command Line
+To create an SPL Token account, for the given mint at a random address and owned
+by the funding account's keypair
+
+```
+spl-token create-account <TOKEN_MINT_ADDRESS>
+```
+
+#### Resulting Output
+```
+Creating account 6VzWGL51jLebvnDifvcuEDec17sK6Wupi4gYhm5RzfkV
+✅ Approved
+Signature: 4JsqZEPra2eDTHtHpB4FMWSfk3UgcCVmkKkP7zESZeMrKmFFkDkNd91pKP3vPVVZZPiu5XxyJwS73Vi5WsZL88D7
+```
+
+### Checking an Account's Balance
+
+#### Command Line
+```
+spl-token balance <TOKEN_ACCOUNT_ADDRESS>
+```
+
+#### Resulting Output
+```
+0
+```
+
+### Token Transfers
+
+For SPL Token transfers to succeed a few prerequisite conditions must be met
+1. The recipient account must exist before the transfer is executed. As described
+in [account creation](#account-creation), SPL Token accounts are *not* explicitly
+created.
+1. Both the sender and recipient accounts must belong to the same mint.  SPL Token
+accounts can only hold one type of SPL token.
+
+#### Command Line
+```
+spl-token transfer 6B199xxzw3PkAm25hGJpjj3Wj3WNYNHzDAnt1tEqg5BN 1 6VzWGL51jLebvnDifvcuEDec17sK6Wupi4gYhm5RzfkV
+```
+
+#### Resulting Output
+```
+Transfer 1 tokens
+  Sender: 6B199xxzw3PkAm25hGJpjj3Wj3WNYNHzDAnt1tEqg5BN
+  Recipient: 6VzWGL51jLebvnDifvcuEDec17sK6Wupi4gYhm5RzfkV
+✅ Approved
+Signature: 3R6tsog17QM8KfzbcbdP4aoMfwgo6hBggJDVy7dZPVmH2xbCWjEj31JKD53NzMrf25ChFjY7Uv2dfCDq4mGFFyAj
+```
+
+### Other Considerations
+
+#### Freeze Authority
+For regulatory compliance reasons, an SPL Token mint may impose "Freeze Authority"
+over all account created for it.  This allows the issuing entity to [freeze](https://spl.solana.com/token#freezing-accounts)
+the assets in a given account at will, rendering the account unusable until thawed.
```

Comment:
Let's mention that this is an optional feature that may be permanently disabled for any given mint, and then explain how to check that in the explorer.

Which brings us to a new explorer feature!    When viewing a Token mint ([example](https://explorer.solana.com/address/TESTpKgj42ya3st2SQTKiANjTBmncQSCqLAZGcSPLGM)) in the explorer, we don't display the Freeze authority at all.  

We could add `spl-token` support for this as well but I feel that if the explorer shows it then we're 99% of the way there and no need to really duplicate the feature in the cli